activejob/README.md

```diff
@@ -1,4 +1,4 @@
-# Active Job -- Make work happen later
+# Active Job – Make work happen later
```

Comment:
Is there any reason to change the title?

Reply:
Hello @kamipo :)
If you will check on ActiveJob README document, you will see double dash there in the output of the file. It's better to have single big dash ( - ) i.e that I suggested. It will be consistent similar to other Rails README documents.

FYI: Well, we always use double dash ( -- ) with in the  README.rdoc kind of files and it converts it to a big single dash but it is a README.md and it does not support double dash ( -- ). It was the reason, I used big single dash there and it's looking perfect :) 

https://github.com/rails/rails/tree/master/activejob

![Screenshot 2019-03-10 at 1 02 18 AM](https://user-images.githubusercontent.com/5468981/54076358-e4287c00-42d0-11e9-8055-be0ef16f4984.png)

Thanks for asking :)